tools/slang-test/slang-test-main.cpp

```diff
@@ -751,11 +751,26 @@ static SlangResult _extractReflectionTestRequirements(const CommandLine& cmdLine
     return SLANG_OK;
 }
 
+static SlangResult _tryUseCPURenderTest(TestContext* context, CommandLine& ioCmdLine)
```

Comment:
Hmm... This is the one bit I'm not too happy with, because I don't like how much `slang-test` ends up needing to inspect the command-line options being passed to individual tests, rather than just keying off of the test type at the start of the line (the original intent was that it would never look at the options).

I'm fine with it as a solution to the immediate problem, but it seems like at some point we'll need to overhaul `slang-test` with a new requirements analysis based on what we've learned from all the new test types/cases we've added over time.